crates/starlight/src/options.rs

```diff
@@ -46,7 +46,7 @@ impl Default for Options {
     fn default() -> Self {
         Self {
             parallel_marking: false,
-            dump_bytecode: false,
+            dump_bytecode: true,
```

Comment:
Can you please set `dump_bytecode` to false again?

Reply:
Oh, sorry. I forget it.